src/__tests__/services/redis/Redis.test.ts

```diff
@@ -0,0 +1,85 @@
+import { createClient } from 'redis';
+import Redis from '../../../services/Redis';
+
+const mockSet = jest.fn((key, value, cb) => cb());
+const mockQuit = jest.fn();
+const mockDel = jest.fn((key, cb) => cb());
+const mockExpire = jest.fn((key, time, cb) => cb());
+const mockExpireAt = jest.fn((key, time, cb) => cb());
+const mockGet = jest.fn((key, cb) => cb(null, 'someData'));
```

Comment:
I think this test might need resetting back to kinda how it was before too, this looks like it was more appropriate for it without the promisifications

Reply:
Just had a look and I think this may be a necessary evil if we want to keep cache.service private.

https://github.com/Pod-Point/cache-js/blob/initial/src/__tests__/services/Redis.test.ts#L18-L22

Reply:
ok yeah no probs